### PR TITLE
Contact me

### PR DESCRIPTION
**Pull request by Gopxfs**
• Added and styled contact me session
• Added footer
• Updated typos

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -148,17 +148,48 @@ <h3 class="work-card-project-title">Multi-Post Stories</h3>
...
+        <div class="contact-me-header">
+          <h2 id="contact-me-title">Contact us</h2>
+          <p id="contact-me-description">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
+        </div>
+        <form action="https://formspree.io/f/xjvlpwgy" method="post">
+          <input type="text" name="name" id="contact-me-name" maxlength="30" placeholder="Full name" required>
+          <input type="email" name="email" id="contact-me-email" placeholder="Input your email" value="example@email.com" required>
+          <textarea name="message" id="contact-me-message" maxlength="500" placeholder="Enter your message" required></textarea>
+          <button class="project-button" id="submit-button" type="submit">Send Message</button>
+        </form>
```
Comment (danmainah) [inline index.html]: - [ ] Kindly make sure to use flex, to position form elements, as specified in the project requirements.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Gopxfs  and @NduatiKagiri , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by williamrolando88 — APPROVED** [review]
Hi @Gopxfs , 

## Your project is complete!
There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

hurra!

## Highlights

- Nice readme file
- Good PR title and description
- Good commit history

## Optional suggestions

- [ ] [Optional] Looks like you are using a fixed-width somewhere in your CSS code, I strongly suggest you replace this fixed-width property, previous reviewers were supposed to restrain you to make this mistake. When you start working in the responsive design you're going to have a lot of trouble dealing with this.

Currently, your project looks like this (picture below) on a desktop screen:
image


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Gopxfs** [comment]
> Hi @Gopxfs ,
> 
> ## Your project is complete!
> There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉
> 
> hurra!
> 
> ## Highlights
> * Nice readme file
> * Good PR title and description
> * Good commit history
> 
> ## Optional suggestions
> * [ ]  [Optional] Looks like you are using a fixed-width somewhere in your CSS code, I strongly suggest you replace this fixed-width property, previous reviewers were supposed to restrain you to make this mistake. When you start working in the responsive design you're going to have a lot of trouble dealing with this.
> 
> Currently, your project looks like this (picture below) on a desktop screen: image
> 
> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Hey! Thank you so much for this reply, I was feeling something was wrong with this but I didn't receive any review explaining what was it so I just kept it. I'm going to try to change it by now.

**Pull request merged**